Moodis/Moodis/Models/User.cs

```diff
@@ -1,22 +1,25 @@
-﻿using System;
+﻿using Moodis.Ui;
+using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Moodis.Feature.Login
 {
     [Serializable]
     class User
     {
-        public string username { get; set; }
-        public string password { get; set; }
+        public String username { get; set; }
+        public String password { get; set; }
+        public List<ImageInfo> imageStats = new List<ImageInfo>();
 
-        public User(string username, string password)
+        public User(String username, String password)
```

Comment:
Isn't string is more preferred than String, according to code style guidelines?

Reply:
will fix probably taken when solving merge conflicts.